src/main/java/org/springframework/hateoas/MediaTypes.java

```diff
@@ -34,4 +35,14 @@
 	 * Public constant media type for {@code application/hal+json}.
 	 */
 	public static final MediaType HAL_JSON = MediaType.valueOf(HAL_JSON_VALUE);
+	
+	/**
+	 * A String equivalent of {@link MediaTypes#HAL_JSON_UTF8}.
+	 */
+	public static final String HAL_JSON_UTF8_VALUE = HAL_JSON_VALUE + ";charset=UTF-8";
```

Comment:
Based upon Spring's MediaType being:

```
/**
 * Public constant media type for {@code application/json;charset=UTF-8}.
 */
public final static MediaType APPLICATION_JSON_UTF8;

/**
 * A String equivalent of {@link MediaType#APPLICATION_JSON_UTF8}.
 */
public final static String APPLICATION_JSON_UTF8_VALUE = APPLICATION_JSON_VALUE + ";charset=UTF-8";
```
...I think the proposed fix is crafted properly.